<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestClass</class>
 <widget class="QMainWindow" name="TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>选择角色</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QComboBox" name="comboBoxBackground"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxExercise">
         <item>
          <property name="text">
           <string>run</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>other</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="btnItemNum">
         <property name="text">
          <string>查看食品数</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnUse">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>吃东西</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxChara"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnBuy">
         <property name="text">
          <string>购买</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2" colspan="2">
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>清空文本框</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="comboBoxChara_Help"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>选择项目</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>选择食物</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>选择项目</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QPushButton" name="btnAddChara">
         <property name="text">
          <string>添加角色</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxItem"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>选择项目</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="comboBoxHouseWork">
         <item>
          <property name="text">
           <string>clean</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cook</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wash</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btnStatus">
         <property name="text">
          <string>查看状态</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QPushButton" name="btnWork">
         <property name="text">
          <string>打工</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="comboBoxWork">
         <item>
          <property name="text">
           <string>banzhuan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>zuoban</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QPushButton" name="btnExercise">
         <property name="text">
          <string>锻炼</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择背景</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QPushButton" name="btnHouseWork">
         <property name="text">
          <string>家务</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>选择项目</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QPushButton" name="btnHelp">
         <property name="text">
          <string>帮助</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2" colspan="2">
        <widget class="QPushButton" name="btnOpenBackground">
         <property name="text">
          <string>打开背景</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
